src/surfaces/filtersRail/filtersRail.jsx

```diff
@@ -1,15 +1,18 @@
-import { map, isEmpty } from 'lodash';
+import { map, isEmpty, isFunction } from 'lodash';
```

Comment:
Nitpick: Should probably be put on separate lines (although I believe ESLINT gives you a pass on this with only two or three).

Reply:
Fixed.

Reply:
Lookin' good.  Resolving.